ImperatorToCK3/Source/Imperator/Countries/Country.h

```diff
@@ -37,20 +42,24 @@ namespace ImperatorWorld
 
 			void setFamilies(const std::map<int, std::shared_ptr<Family>>& newFamilies) { families = newFamilies; }
 
+			void registerCK3Title(const std::pair<std::string, std::shared_ptr<CK3::Title>>& theTitle) { ck3Title = theTitle; }
+
 		private:
 			void registerKeys();
 
 			int countryID = 0;
 			std::string tag;
 			std::string name;
 			std::string flag;
+			std::optional<int> capital;
```

Comment:
on second thought optional<int> means nothing. if you do if(capital) it'll always be true if not zero.